orka-utils.c

```diff
@@ -130,6 +130,24 @@ orka_iso8601_to_unix_ms(char *timestamp, size_t len, void *p_data)
   return 1; // SUCCESS
 }
 
+int
+orka_unix_ms_to_iso8601(char *str, size_t len, void *p_data)
+{
+  uint64_t *p_timestamp = (uint64_t*)p_data;
+  ASSERT_S(NULL != p_timestamp, "No timestamp provided by user");
+  uint64_t timestamp = *p_timestamp;
+
+  time_t seconds = timestamp / 1000;
+  int millis = timestamp % 1000;
+
+  struct tm *tm = localtime(&seconds);
+
+  return snprintf(str, len,
+    "%d-%.2d-%dT%.2d:%.2d:%.2d.%.3dZ", // ISO-8601 complete format
+    tm->tm_year + 1900, tm->tm_mon + 1, tm->tm_mday, // Date
+    tm->tm_hour, tm->tm_min, tm->tm_sec, millis); // Time
+}
+
 int
```

Comment:
this function has a special requirement: If I call it with this
```
orka_unix_ms_to_iso8601(NULL, 0, void *p_data)
```
It should return the number of bytes needed to convert p_data to string without including the null terminating byte. 
I think you can fix it in the next commit.

Reply:
hmm, actually it works because `snprintf` will do the job.  You are good.